dotCMS/src/main/java/com/dotmarketing/common/reindex/ReIndexerStatus.java

```diff
@@ -0,0 +1,98 @@
+package com.dotmarketing.common.reindex;
+
+import com.dotcms.content.elasticsearch.business.ESIndexAPI;
+import java.io.Serializable;
+import java.util.Hashtable;
+import java.util.Map;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.content.elasticsearch.business.ContentletIndexAPIImpl;
+import com.dotcms.content.elasticsearch.business.IndiciesInfo;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)